streamz/tests/test_kafka.py

```diff
@@ -152,6 +171,7 @@ def test_from_kafka_thread():
         stream = Stream.from_kafka([TOPIC], ARGS)
         out = stream.sink_to_list()
         stream.start()
+        sleep(2)
```

Comment:
Why this sleep?  In general sleeps can be troublesome because:

1.  They make tests take a long time if they are overly conservative
2.  If the system running the tests is abnormally slow then they can fail (Travis is often abnormally slow)

I'm also concerned about using `time.sleep` here in an async test.  This is usually an anti-pattern.

Reply:
Yes, I totally agree. I'll take some time to come up with a better solution.